<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>auth</class>
 <widget class="QWidget" name="auth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Authorization</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Авторизация</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineLogin">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>100</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 255);
color: rgb(41, 0, 62);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePass">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>170</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);
color: rgb(0, 74, 0);
</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAuth">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Войти</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>213</x>
     <y>70</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Логин</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Exit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
